### ISSUE TITLE
Is the ADUM6404 VDDL pin ..wrong voltage

### ISSUE THREAD

**Issue by koenver** [digitaldex/piMeter#4]
I think the PCB has a wrong connection at the ADUM6404 VDDL pin.
Now it is connected to the +5V_ISO..so the output of the ADUM6404 VOA..VOD is between 0 and 5 V, that's bad for the raspberry.
When I connect the VDDL pin of the ADUM6404 at +3V3_ISO the logic levels are normal between 0 and 3.3V.  ...I tested this and this was correct.

greetings
Koen

**Comment by digitaldex** [comment]
You are right. The ADUM Data Channels in this configuration work with 5V. I was thinking the gpio pins on the pi are 5V tolerant but in fact they are not. 

But for your workaround check the ADUM Datasheet:

> Page 16 Table 25 Data Channel Supply Voltage, 3.0 V to 5.5 V. Pin 1 and Pin 7 must be connected to the same external voltage source.

This is not the case if you use 3.3V on the VDDL Pin. To be safe use a Voltage Divider for CF1, CF2, IRQ1 and IRQ2 if you realy need those pins. i havent used any of these right now.

greetings and thanks for the input

**Comment by koenver** [comment]
Sorry, but I think AD devices means with the same voltage source..the same voltage group... at the same side of the chip.  According to the datasheet VDD1 (5V) with GND1 and with the 3V3 must also have GND1 as ground.
It works perfect..I lifted the VDDL pin and soldered a wire to it to 3V3 (from raspberry side)...Than I tested with 3V3_ADE (high and low at the input pins) and at the output pins was a nice 3V3_raspberry or zero (GND1)..
Its a bit confusing because the AD spreadsheet calls the left side" ISO" and  digitaldex the right side of the chip "ISO"

**Comment by koenver** [comment]
No I am using your basic design and mixed with the original evaluation board...so not the current headphone connectors, also the rogowski components are back,  Aswell a raspberry as a ESP32 (D4 micro) can be connected and a GPS (tx,rx,pps).
With jumpers it's selectable between CF1/CF3 and CF2/CF4  (CF1 and CF2 are only needed for calibration), while CF4 gives the zero crossing... 
But I also forgot in case of the ESP32 that normally the Raspberry generates the 3V3 from the 5V :)..
but it a litte micro pcb in the RP slot created the 3V3 also :)....and  I had the ADUM6404 VDDL mistake..
If every thing is succesfull and I fix these errors I will publish the PCB and Eagle files.

koen

**Comment by digitaldex** [comment]
Okay thanks for the info. I will fix this soon and close the issue when i am done
